src/core-asyncpipeline/AsyncPipeline/AsyncPipeline.T/Pipe/Pipe.cs

```diff
@@ -0,0 +1,26 @@
+#nullable enable
+
+using System.Threading.Tasks;
+
+namespace System
+{
+    partial struct AsyncPipeline<T>
+    {
+        public AsyncPipeline<TResult> Pipe<TResult>(Func<T, TResult> map)
```

Comment:
Let's consider to rename `map[Async]` strategy to `pipe[Async]` in `AsyncPipeline` (but not in `AsyncResultFlow`).

And that's the reasons why:
- On the first hand, in general map looks ok for the Pipe function.
- On the second hand, `AsyncPipeline` is in the same context with `AsyncResultFlow`.
- And in `AsyncResultFlow` map has some more specific semantics. E.g., the result flow has `Forward` function with `next` and `map` strategies, and the `map` means only some projection only, not a pipe process. Also, the result flow has `Map` function with the `map`/`map` strategies that also mean projection only.
- So, whether `map` is used for `Pipe` in the simple `[Async]Pipeline` (i.g., for some pipe process, not just for simple projection), it would be ambiguous between the pipeline and the result flow.
- And I guess some neutral and at the same time general-purposed term should be used. It seems, `pipe` strategy looks ok. It may mean some general pipe process, including forwarding, filtering, recovering, projection (mapping), etc., as cases for the general-purposed pipeline that has the only `Pipe` method.

Reply:
Done